src/Symfony/Component/Serializer/Normalizer/PropertyNormalizer.php

```diff
@@ -84,6 +84,21 @@ protected function isAllowedAttribute($classOrObject, $attribute, $format = null
                 return false;
             }
         } catch (\ReflectionException $reflectionException) {
+            $className = is_string($classOrObject) ? $classOrObject : get_class($classOrObject);
+            $class = new \ReflectionClass($className);
+
+            while ($parent = $class->getParentClass()) {
+                try {
+                    $reflectionProperty = new \ReflectionProperty($parent->getName(), $attribute);
```

Comment:
As you have an instance of `ReflectionClass` available, maybe can you just call `getProperty`?